lib/dids/bls_test.go

```diff
@@ -81,46 +81,61 @@ func TestFullCircuitFlow(t *testing.T) {
 	finalCircuit, err := partialCircuit.Finalize()
 	assert.NoError(t, err)
 
-	// verify the aggregated signature in the final circuit
-	verified, err := finalCircuit.Verify()
+	// get the aggregated signature and bit vector from the circuit
+	aggSigEncoded, err := finalCircuit.AggregatedSignature()
+	assert.NoError(t, err)
+	bvEncoded, err := finalCircuit.BitVector()
 	assert.NoError(t, err)
-	assert.True(t, verified)
 
-	// verify the pub keys of the members match the final aggregated pub key in the circuit
-	didsToVerify := []dids.BlsDID{did1, did2}
-	verifiedPubKeys, err := finalCircuit.VerifyDIDs(didsToVerify)
+	// now call Verify with the aggregated signature and bit vector
+	verified, includedDIDsVerified, err := finalCircuit.Verify(aggSigEncoded, bvEncoded)
 	assert.NoError(t, err)
```

Comment:
This still seems weird to use. I think some more design around the interface would be useful here.